handbook/company/pricing-features-table.yml

```diff
@@ -1391,6 +1401,7 @@
   pricingTableCategories: [Integrations]
   comingSoonOn: 2024-06-30
   tier: Free
+  isExperimental: yes
```

Comment:
```suggestion
```

Hasn't shipped yet.

Also, it looks like we have an outdated `comingSoonOn` for this feature.

I'll open a separate PR for this.